tools/validate/validate.cpp

```diff
@@ -192,7 +187,7 @@ void scan_file(
         static_assert(ranges::is_sorted(checked_extensions));
 
         if (ranges::binary_search(checked_extensions, filepath.extension().wstring())) {
-            validation_failure(any_errors, filepath, L"file contains %zu lines with more than %zu columns.\n",
+            validation_failure(any_errors, filepath, "file contains {} lines with more than {} columns.",

```

Comment:
No change requested: This fixes a mistake that was introduced by #2671. `validation_failure()` prints a newline, so we don't want a second one here.